vendor/github.com/mongodb/mongo-tools-common/db/namespaces.go

```diff
@@ -120,16 +77,6 @@ func GetCollections(database *mongo.Database, name string) (*mongo.Cursor, error
 	return cursor, nil
 }
 
-func getCollectionsPre28(database *mgo.Database, name string) (*mgo.Iter, error) {
```

Comment:
this was deleted because GODRIVER-492 has been implemented, right? should we also delete the comment above on `GetCollections`?

Reply:
ditto comment above